front-end/src/components/RegistrationForm/index.jsx

```diff
@@ -130,7 +131,7 @@ const RegistrationForm = () => {
 
   const handleChangeActive = (tab) => {
     setActiveTab(tab);
-    setData((prev) => ({ ...prev, role: `${tab === 0 ? 'student' : 'teacher'}` }));
+    setData((prev) => ({ ...prev, role: `${tab === 0 ? 'Student' : 'Teacher'}` }));
```

Comment:
use student role constant

Reply:
@andriydzhrvts updated to role constants